docs/usage.md

```diff
@@ -0,0 +1,94 @@
+
+`sample, fastq_1, fastq_2`
+Neither the `sample` or `fastq*` columns may contain spaces, and the `fastq` columns contains the path(s) to your fastq file(s).
+
+If you are inputting Illumina reads, the samplesheet looks like the following:
+
+```
+sample,fastq_1,fastq_2
+comm_1,https://github.com/Arcadia-Science/test-datasets/raw/main/metagenomics/illumina/comm_1_subsampled_1.fq.gz,https://github.com/Arcadia-Science/test-datasets/raw/main/metagenomics/illumina/comm_1_subsampled_2.fq.gz
+vir_1,https://github.com/Arcadia-Science/test-datasets/raw/main/metagenomics/illumina/vir_1_subsampled_1.fq.gz,https://github.com/Arcadia-Science/test-datasets/raw/main/metagenomics/illumina/vir_1_subsampled_2.fq.gz
+```
+
+If you are inputting Nanopore reads, the samplesheet looks like the following:
+
+```
+sample,fastq_1,fastq_2
+om,https://github.com/Arcadia-Science/test-datasets/raw/main/metagenomics/ont/om_subset_reads.fq.gz,
+el,https://github.com/Arcadia-Science/test-datasets/raw/main/metagenomics/ont/el_subset_reads.fq.gz,
+```
+
+Note that even for Nanopore reads which the input is in a single fastq file, still include the third column for `fastq_2` - the pipeline for processing Nanopore reads will ignore this and process your Nanopore reads that are in a single fastq file.
+
+## Running the pipeline
+
+A typical command for running the pipeline looks like:
+` nextflow run Arcadia-Science/metagenomics --input <SAMPLESHEET.csv> --outdir <OUTDIR> --platform <illumina|nanopore> -profile docker`
+
+This will launch the pipeline with the `docker` configuration profile. See below for more information about profiles. The options for `--platform` are either `illumina` or `nanopore` depending on your sequencing file input. Therefore, you cannot run both Illumina and Nanopore sequencing files in the same run - instead submit separate runs. The pipeline will create the following files and directories in your working directory:
+
+```
+work                # Directory containing the nextflow working files
+<OUTDIR>            # Finished results in specified location (defined with --outdir)
+.nextflow_log       # Log file from Nextflow
+# Other nextflow hidden files, eg. history of pipeline runs and old logs.
+```
+
+### Updating the pipeline
+
+When you run the above command, Nextflow automatically pulls the pipeline code from GitHub and stores it as a cached version. When running the pipeline after this, it will always use the cached version if available - even if the pipeline has been updated since. To make sure you are running the latest version of the pipeline, regularly update the cached version with:
+`nextflow pull Arcadia-Science/metagenomics`
+
+## Pipeline arguments
+
+Below lists several arguments to configure core parts of how Nextflow is run and pipeline-specific arguments you can modify.
+
+### Core Nextflow arguments
+
+These options are part of Nextflow and use a single hyphen.
```

Comment:
can you run something like `nextflow -h` to see all of the available parameters? below you describe a subset of useful ones, only some of which are required right?